site/docs/main/configuration.md

```diff
@@ -189,6 +190,31 @@ The gateway configuration block is used to configure which service-apis Gateway
 |------------|-----|----------|-------------|
 | name | string | contour | This field specifies the name of a Gateway.  |
 | namespace | string | projectcontour | This field specifies the namespace of a Gateway.  |
+
+### Policy Configuration
+
+The Policy configuration block can be used to configure default policy values
+that are set if not overridden by the user.
+
+The `request-headers` field is used to rewrite headers on a HTTP request, and
+the `response-headers` field is used to rewrite headers on a HTTP response.
+
+| Field Name | Type| Default  | Description |
+|------------|-----|----------|-------------|
+| request-headers | HeaderPolicy | none | The default request headers set or removed on all service routes if not overridden in the object |
+| response-headers | HeaderPolicy | none | The default response headers set or removed on all service routes if not overridden in the object |
+{: class="table thead-dark table-bordered"}
+<br>
+
+#### HeaderPolicy
+
+The `set` field sets an HTTP header value, creating it if it doesn't already exist but not overwriting it if it does.
+The `remove` field removes an HTTP header.
+
+| Field Name | Type| Default  | Description |
+|------------|-----|----------|-------------|
+| set | map[string]string | none | Map of headers to set on all service routes if not overridden in the object |
+| remove | []string | none | List of headers to remove on all service routes if not overridden in the object |
```

Comment:
similarly, it'd be helpful to clarify that this set of removes is appended to an individual proxy's list of removes, i.e. an individual proxy can't remove an item from the global list of removes.

Reply:
Agreed, added a note.